
real	0m48.206s
user	0m42.384s
sys	0m2.954s

real	2m22.279s
user	2m13.833s
sys	0m6.116s

real	0m47.399s
user	0m43.066s
sys	0m2.320s

real	2m20.639s
user	2m12.819s
sys	0m5.528s

real	0m35.836s
user	0m32.820s
sys	0m1.147s

real	2m18.106s
user	2m13.802s
sys	0m1.913s

real	0m47.854s
user	0m43.748s
sys	0m2.471s

real	2m22.190s
user	2m13.880s
sys	0m5.935s

real	0m47.677s
user	0m43.564s
sys	0m2.334s

real	2m22.800s
user	2m18.469s
sys	0m2.171s
Traceback (most recent call last):
  File "/home/thiago.esterci/2D_imune_edema_pinn/edo_pinn_model.py", line 176, in <module>
    Cp = pk.load(f)
EOFError: Ran out of input

real	0m2.065s
user	0m1.377s
sys	0m0.189s

real	2m17.858s
user	2m12.347s
sys	0m3.343s

real	0m46.340s
user	0m43.232s
sys	0m1.403s

real	2m18.582s
user	2m12.353s
sys	0m3.722s

real	0m46.980s
user	0m43.183s
sys	0m2.067s

real	2m17.650s
user	2m13.401s
sys	0m1.954s

real	0m44.924s
user	0m42.016s
sys	0m1.348s

real	2m0.990s
user	1m57.198s
sys	0m1.921s

real	0m44.965s
user	0m42.004s
sys	0m1.386s

real	2m6.952s
user	2m2.872s
sys	0m1.852s
